website/content/docs/secrets/aws.mdx

```diff
@@ -344,7 +349,52 @@ lease_duration 	60m0s
 lease_renewable	false
 access_key     	ASIAJYYYY2AA5K4WIXXX
 secret_key     	HSs0DYYYYYY9W81DXtI0K7X84H+OVZXK5BXXXX
-security_token 	AQoDYXdzEEwasAKwQyZUtZaCjVNDiXXXXXXXXgUgBBVUUbSyujLjsw6jYzboOQ89vUVIehUw/9MreAifXFmfdbjTr3g6zc0me9M+dB95DyhetFItX5QThw0lEsVQWSiIeIotGmg7mjT1//e7CJc4LpxbW707loFX1TYD1ilNnblEsIBKGlRNXZ+QJdguY4VkzXxv2urxIH0Sl14xtqsRPboV7eYruSEZlAuP3FLmqFbmA0AFPCT37cLf/vUHinSbvw49C4c9WQLH7CeFPhDub7/rub/QU/lCjjJ43IqIRo9jYgcEvvdRkQSt70zO8moGCc7pFvmL7XGhISegQpEzudErTE/PdhjlGpAKGR3d5qKrHpPYK/k480wk1Ai/t1dTa/8/3jUYTUeIkaJpNBnupQt7qoaXXXXXXXXXX
+session_token 	AQoDYXdzEEwasAKwQyZUtZaCjVNDiXXXXXXXXgUgBBVUUbSyujLjsw6jYzboOQ89vUVIehUw/9MreAifXFmfdbjTr3g6zc0me9M+dB95DyhetFItX5QThw0lEsVQWSiIeIotGmg7mjT1//e7CJc4LpxbW707loFX1TYD1ilNnblEsIBKGlRNXZ+QJdguY4VkzXxv2urxIH0Sl14xtqsRPboV7eYruSEZlAuP3FLmqFbmA0AFPCT37cLf/vUHinSbvw49C4c9WQLH7CeFPhDub7/rub/QU/lCjjJ43IqIRo9jYgcEvvdRkQSt70zO8moGCc7pFvmL7XGhISegQpEzudErTE/PdhjlGpAKGR3d5qKrHpPYK/k480wk1Ai/t1dTa/8/3jUYTUeIkaJpNBnupQt7qoaXXXXXXXXXX
+```
+
+### STS Session Tokens
+
+The `session_token` credential type is used to generate short-lived credentials under the root config.
+To create these with Vault and AWS, you must configure Vault to use IAM user credentials. AWS does not
+allow temporary credentials, like those from an IAM instance profile, to be used when generating session tokens.
+
+<Important>
+  STS session tokens inherit any and all permissions granted to the user configured in `aws/config/root`.
+  In this expample, the `temp_user` role will obtain a policy with the same `ec2:*` permissions as the
+  root config. For this reason, assigning a role or policy is disallowed for this credential type.
+</Important>
```

Comment:
```suggestion
<Warning>
  
  STS session tokens inherit any and all permissions granted to the user configured in `aws/config/root`.
  In this expample, the `temp_user` role will obtain a policy with the same `ec2:*` permissions as the
  root config. For this reason, assigning a role or policy is disallowed for this credential type.
  
</Warning>
```

There's a bug in the dev portal where markdown in aside text won't render properly unless there's whitespace before/after the aside tag. Also, for some reason, we don't support the warning tag anymore, just "important":

![image](https://github.com/hashicorp/vault/assets/62406755/bbb2a3cc-3534-49dc-ab77-3076d9964780)

Reply:
Thanks!